src/Sulu/Bundle/DocumentManagerBundle/Command/FixturesLoadCommand.php

```diff
@@ -113,7 +113,13 @@ function(BundleInterface $bundle) {
                 $this->kernel->getBundles()
             );
 
-            $directories[] = $this->kernel->getRootDir();
+            if (method_exists($this->kernel, 'getRootDir')) {
+                $directories[] = $this->kernel->getRootDir();
+            }
+
+            $directories[] = $this->kernel->getProjectDir();
+            $directories[] = $this->kernel->getProjectDir() . '/src';
```

Comment:
This is also part of another PR, isn't it?

Reply:
Removed from this PR